demos/mri_reconstruction_demo/python/mri_reconstruction_demo.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import argparse
+import time
+import logging
+import sys
+
+import numpy as np
+import cv2 as cv
+from openvino.inference_engine import IECore
+
+logging.basicConfig(format='[ %(levelname)s ] %(message)s', level=logging.INFO, stream=sys.stdout)
+logger = logging.getLogger('mri_reconstruction_demo')
+
+def kspace_to_image(kspace):
+    assert(len(kspace.shape) == 3 and kspace.shape[-1] == 2)
+    fft = cv.idft(kspace, flags=cv.DFT_SCALE)
+    img = cv.magnitude(fft[:, :, 0], fft[:, :, 1])
+    return cv.normalize(img, dst=None, alpha=255, beta=0, norm_type=cv.NORM_MINMAX, dtype=cv.CV_8U)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='MRI reconstrution demo')
```

Comment:
as I can see the preferable style of python demos to not use code directly in global space (under if __name__ == '__main__' ), please wrap it to main function